amundsen_application/static/css/_typography-custom.scss

```diff
@@ -0,0 +1 @@
+// This file is intentionally left blank and should be overwritten by in the build process.
```

Comment:
nit: I'm in favor of a more explicit explanation like `...should be used to add new typography or overwrite defaults from _typography-default.scss`. For example if someone wants to fork our repo and override files that way, I find instruction of having some "build process" confusing.